sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/ecert-full-time-feedback-integration.scheduler.ts

```diff
@@ -13,7 +13,7 @@ export class FullTimeECertFeedbackIntegrationScheduler extends BaseScheduler<voi
     @InjectQueue(QueueNames.FullTimeFeedbackIntegration)
     schedulerQueue: Queue<void>,
     queueService: QueueService,
-    private readonly eCertFileHandler: ECertFileHandler,
+    private readonly fullTimeECertFileHandler: FullTimeECertFileHandler,
```

Comment:
Should we access the file handler though base class like this?

![image](https://github.com/bcgov/SIMS/assets/54600590/c7f83f78-a775-4ec9-9c20-db868da4853c)

Reply:
If there is a benefit I would support it but I do not see any, please clarify if I am missing something.
The injection would happen in one way or another, so we have 2 ways to achieve the same. 
I see the usage of the `@Inject` for objects like `Connection` or `DataSource` where multiple can exist and they do not have individual implementations.